ozaria/site/components/common/LayoutChrome.vue

```diff
@@ -9,25 +11,104 @@
       chromeOn: {
         type: Boolean,
         default: false
+      },
+
+      optionsClickHandler: {
+        type: Function
+      },
+
+      restartClickHandler: {
+        type: Function
+      }
+    },
+
+    computed: {
+      ...mapGetters({
+        isSoundOn: 'layoutChrome/isSoundOn',
+        getCurrentCampaignId: 'layoutChrome/getCurrentCampaignId',
+        getCourseInstanceId: 'layoutChrome/getCurrentCourseInstanceId'
+      }),
+
+      mapLink () {
+        if (me.isSessionless() || !this.getCurrentCampaignId || !this.getCourseInstanceId) {
+          return '/teachers/courses'
+        }
+        return `/ozaria/play/${this.getCurrentCampaignId}?course-instance=${this.getCourseInstanceId}`
+      }
+    },
+
+    methods: {
+      ...mapActions('layoutChrome', ['toggleSoundAction']),
+
+      clickOptions () {
+        if (this.optionsClickHandler) {
+          this.optionsClickHandler()
```

Comment:
We should use Vue's eventing system ($emit) for this and click restart unless we have a good reason to go with handlers directly.

Reply:
@jmif I agree that this is an anti pattern, when an emitter should be used. Should displaying these buttons be a boolean prop?